package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "casbin-redis-adapter",
+  "version": "0.0.1",
+  "main": "lib/cjs/index.js",
+  "typings": "lib/cjs/index.d.ts",
+  "module": "lib/esm/index.js",
+  "scripts": {
+    "postpack": "run-s clean",
+    "build": "run-s clean && run-p build:*",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:esm": "tsc -p tsconfig.esm.json",
+    "clean": "rimraf lib",
+    "test": "jest"
+  },
+  "homepage": "https://casbin.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/node-casbin/redis-adapter.git"
+  },
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "https://www.apache.org/licenses/LICENSE-2.0"
+    }
+  ],
+  "dependencies": {
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.12.5",
+    "casbin": "^5.9.0",
+    "jest": "^27.0.5",
+    "redis": "^3.1.1"
```

Comment:
move `casbin` and `jest` to `devDependencies`, and add `casbin` as `peerDependencies`.